atd-vze/src/views/Crashes/Crash.js

```diff
@@ -131,7 +131,9 @@ function Crash(props) {
     investigator_narrative_ocr: investigatorNarrative,
   } = !!data?.atd_txdot_crashes[0] ? data?.atd_txdot_crashes[0] : {};
 
-  const crashRecord = { crash: crashData?.crashes?.[0] || {} };
+  const crashRecord = {
+    crash: crashData?.crashes?.[0] || { crash_injury_metrics_view: {} },
+  };
```

Comment:
addresses the destructing bug that Chia flagged: https://github.com/cityofaustin/atd-vz-data/pull/1470/files#r1630268054